src/components/Home.jsx

```diff
@@ -1,62 +1,108 @@
-    return (
-        <div>
-            <Navbar />
-            <div className="body">
-            <form>
-                <div className="form-group">
-                    <textarea rows="6" cols="200" className="text-field" placeholder="What's happening? Max. 280 characters."/>
-                </div>
-                <button type="submit" className="btn btn-primary tweet-btn">Tweet</button>
-                <p className="chars"> 38 characters</p>
-                <div className="tweets-container">
-                   <p className="filter-by ml-0">Filter-by</p>
-                   <button type="submit" className="btn btn-default tag-button">#Learntech</button>
-                   <div className="tweets">
-                   <img src={Avatar} alt="Avatar" className="avatar" />
-                   <span className="tweet-body">
-                   <span className="tweet-time">2 hours ago</span>
-                    <h6 className="tweet-user">Maya Neria</h6>   
-                    <p class="main-tweet">I love #regle so much - everyone should use it! #learntech</p>
-                   </span>
-                   </div>
-                   <div className="tweets">
-                   <img src={Avatar} alt="Avatar" className="avatar" />
-                   <span className="tweet-body">
-                   <span className="tweet-time">2 hours ago</span>
-                    <h6 className="tweet-user">Maya Neria</h6>   
-                    <p class="main-tweet">I love #regle so much - everyone should use it! #learntech</p>
-                   </span>
-                   </div>
-                   <div className="tweets">
-                   <img src={Avatar} alt="Avatar" className="avatar" />
-                   <span className="tweet-body">
-                   <span className="tweet-time">2 hours ago</span>
-                    <h6 className="tweet-user">Maya Neria</h6>   
-                    <p class="main-tweet">I love #regle so much - everyone should use it! #learntech</p>
-                   </span>
-                   </div>
-                   <div className="tweets">
-                   <img src={Avatar} alt="Avatar" className="avatar" />
-                   <span className="tweet-body">
-                   <span className="tweet-time">2 hours ago</span>
-                    <h6 className="tweet-user">Maya Neria</h6>   
-                    <p class="main-tweet">I love #regle so much - everyone should use it! #learntech</p>
-                   </span>
-                   </div>
+class Index extends Component {
+    constructor (props){
+        super(props)
+        this.state = {
+            value: 280,
+            disabled: false,
+            text_value: ''
+        }
+
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChange(event){
+        this.setState({value: 280-event.target.value.length, text_value: event.target.value})
```

Comment:
In my opinion, this side effect should be handled outside `setState`

Reply:
Thanks. I have passed it in a variable. Fixed now. Kindly confirm